src/MidiConversions.elm

```diff
@@ -7,8 +7,28 @@ import Midi.Generate exposing (recording)
 import BinaryBase64 exposing (encode)
 
 
-type alias MidiNumber =
-    Int
+type alias ContentType =
+    String
+
+
+type alias ContentTransferEncoding =
+    String
+
+
+type alias MidiAsBase64EncodedString =
+    String
+
+
+type alias MidiEncoding =
+    { contentType : ContentType
+    , contentTransferEncoding : ContentTransferEncoding
+    , midiData : MidiAsBase64EncodedString
+    }
+
+
+toDataString : MidiEncoding -> String
+toDataString midi =
```

Comment:
I am not sure what the best way to deconstruct a record is. Maybe there is a a 'let' expression or something.